pkg/pager/model.go

```
@@ -0,0 +1,377 @@
+func (m *model) Init() tea.Cmd {
+	return nil
+}
+
+var ansiRegex = regexp.MustCompile(`\x1b\[[0-9;?]*[ -/]*[@-~]`)
+
+// StripANSI removes ANSI escape sequences (like terminal colors) from input.
+func StripANSI(input string) string {
+	return ansiRegex.ReplaceAllString(input, "")
+}
+
+func (m *model) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
```

golangci-lint (1.64.8)

[error] cyclomatic: function (*model).Update has cyclomatic complexity 11 (> max enabled 10)

(revive)